### PR TITLE
ovnkube.sh: make northd backoff interval configurable

### PR DESCRIPTION
**Pull request by cathy-zhou**
**- What this PR does and why is it needed**

**- Special notes for reviewers**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

**Comment by tssurya** [comment]
could we please get some more description on the PR message around the context of this change?

File: dist/images/ovnkube.sh [hunk 1]
```diff
@@ -943,9 +948,11 @@ local-nb-ovsdb() {
   wait_for_event attempts=3 process_ready ovnnb_db
   echo "=============== nb-ovsdb (unix sockets only) ========== RUNNING"
 
-  # Let ovn-northd sleep and not use so much CPU
-  ovn-nbctl set NB_Global . options:northd-backoff-interval-ms=300
-  echo "=============== nb-ovsdb ========== reconfigured for northd backoff"
+  [[ "true" == ${ovn_enable_northd_backoff_interval} ]] && {
+    # Let ovn-northd sleep and not use so much CPU
```
[the remark this answers is not in the document]
Comment (cathy-zhou) [reply dist/images/ovnkube.sh:952]: downstream we are still use 23-06 ovn, and this option is supported starting from ovn 23-09. Also, we discussed internally, and we were not convinced yet that this change is valueble to our usage.

[the remark this answers is not in the document]
Comment (cathy-zhou) [reply dist/images/ovnkube.sh:952]: let me update the MR to specifiy the northd-interval instead, and 0 would mean not set the setting.

[the remark this answers is not in the document]
Comment (cathy-zhou) [reply dist/images/ovnkube.sh:957]: done

**Review by trozet — DISMISSED** [review]

**Review by trozet — APPROVED** [review]

**Pull request merged**